app/api/datastore/base.rb

```diff
@@ -12,7 +12,7 @@ class Base < Grape::API
     end
 
     rescue_from ActiveRecord::RecordNotUnique do
-      error!({ status: 400, error: 'Record not unique' }, 400)
+      error!({ status: 409, error: 'Record not unique' }, 409)
```

Comment:
Wondering if it would be safer to raise a specific exception. Like we do with the `Errors::Already***`
Just thinking if there are other places where `ActiveRecord::RecordNotUnique` can be raised maybe is best to have this one as a catch-all unless a specific exception is raised.